onnxruntime/python/tools/tensorrt/perf/benchmark.py

```diff
@@ -854,25 +827,25 @@ def get_test_data(fp16, test_data_dir, all_inputs_shape):
     return inputs, ref_outputs
 
 def create_session(model_path, providers, session_options):
-
     logger.info(model_path)
     try:
         session = onnxruntime.InferenceSession(model_path, providers=providers, sess_options=session_options)
-
         return session
-    except:
-        logger.info("Use symbolic_shape_infer.py")
-
+    except Exception as e:
+        if "shape inference" in str(e):
+            logger.info("Using model from symbolic_shape_infer.py")
 
-    new_model_path = model_path[:].replace(".onnx", "_new_by_trt_perf.onnx")
-    exec = os.environ["SYMBOLIC_SHAPE_INFER"]
-    logger.info(exec)
+            new_model_path = model_path[:].replace(".onnx", "_new_by_trt_perf.onnx")
+            exec = os.environ["SYMBOLIC_SHAPE_INFER"]
 
-    if not os.path.exists(new_model_path):
-        subprocess.run("python3 " + exec +" --input " + model_path + " --output " + new_model_path + " --auto_merge", shell=True, check=True)
-    session = onnxruntime.InferenceSession(new_model_path, providers=providers, sess_options=session_options)
-
-    return session
+            if not os.path.exists(new_model_path):
+                p = subprocess.run("python3 " + exec + " --input " + model_path + " --output " + new_model_path + " --auto_merge", shell=True, check=True, )
+                logger.info(p) 
+            
+            session = onnxruntime.InferenceSession(new_model_path, providers=providers, sess_options=session_options)
+            return session
+        else:
+            raise Exception(e) 
 
 def run_onnxruntime(args, models):
```

Comment:
before standalone_trt was run --trtexec args existed in the same benchmark process with trt ep. This caused issues with logging/ outputting results. Now standalone_trt runs as its own benchmark process, especially so we can catch errors properly.